cpp/include/Ice/SSL/ServerAuthenticationOptions.h

```diff
@@ -173,7 +173,7 @@ namespace Ice::SSL
         /// @param trust The trust object that contains the client's certificate chain.
         /// @param info The connection info object that provides additional connection-related data. The
         /// `ConnectionInfo` type is an alias for the platform-specific connection info class.
-        /// @return true if the certificate chain is valid and the connection should proceed; false if the certificate
+        /// @return `true` if the certificate chain is valid and the connection should proceed; false if the certificate
```

Comment:
There's even more in this file that I'm not pointing out.
```suggestion
        /// @return `true` if the certificate chain is valid and the connection should proceed; `false` if the certificate
```